### PR TITLE
TK257 - 

### PR DESCRIPTION
**Pull request by karuna835**
new dropdown field to select lead owner in create enrollment popup (use the same logic in search payment).

### CODE REVIEW

File: app/apis/webapi/backend/dash/get.leadowner.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+	// Auth Check - Expecting Session Only !
+	if ((!auth_session_is_logged()) || (!auth_session_is_allowed("dash"))) {
+		header("HTTP/1.1 401 Unauthorized");
+		die();
+	}
+
+	//load_module("course");
+
+    $data = db_query("SELECT user_id, name FROM `user` where find_in_set(15, replace(roles, ';', ','))");
```
Comment (kailasb) [inline app/apis/webapi/backend/dash/get.leadowner.php:36]: add constant for 15

File: app/apis/webapi/backend/dash/package.query.php [hunk 2]
```diff
@@ -380,8 +382,17 @@ function where_clause($name, $operator, $value, &$where, $joiner="") {
...
-
+        $lead_owner_data = db_query("select name from user where user_id = ".$package["lead_owner_id"]."");
+		if(!empty($lead_owner_data))
+		{
+			$package["lead_owner"] = $lead_owner_data[0]['name'];
+		}
+		else
+		{
+			$package["lead_owner"] = "";
+		}
```
Comment (kailasb) [inline app/apis/webapi/backend/dash/package.query.php:395]: check user_id & lead_owner_id should same so no need to execute extra query in loop

File: app/templates/jaws/backend/dash/assets/js/controllers/payment.edit.js [hunk 3]
```diff
@@ -50,8 +50,10 @@ angular.module('jaws')
         });
 
         $scope.courses = courses;
-        // console.log(courses);
-
+        //console.log(leadowner[0]);
+		//TK257 - assign lead owner to dropdown.
+        $scope.leadowner = leadowner[0];
+		$scope.lead_owner = leadowner.user_id;
```
Comment (kailasb) [inline app/templates/jaws/backend/dash/assets/js/controllers/payment.edit.js:56]: - added empty check condition
- Give key name in leader owner array

File: app/templates/jaws/backend/dash/assets/js/controllers/payment.edit.js [hunk 4]
```diff
@@ -969,14 +971,16 @@ angular.module('jaws')
                     'data_user_state': $scope.user_state,
                     'expire_date': $scope.expiry,
                     'course_start_date': $scope.course_start,
-		    'application_number': $scope.user.applicationNumber, // TK146
+		    'application_number': $scope.user.applicationNumber,
+				// TK146
```
Comment (kailasb) [inline app/templates/jaws/backend/dash/assets/js/controllers/payment.edit.js:975]: remove unwanted lines

File: app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js [hunk 5]
```diff
@@ -221,6 +220,9 @@ angular.module('jaws')
             desc: '',
             detail: ''
         };
+		//TK257 - assign lead owner to dropdown.
+		$scope.leadowner = leadowner[0];
+		$scope.lead_owner = leadowner.user_id;
```
Comment (kailasb) [inline app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js:225]: add empty check & make array key value pair

**Review by kailasb — APPROVED** [review]
check given feedback

**Review by kailasb — CHANGES_REQUESTED** [review]
check given feedback

File: app/apis/webapi/backend/dash/get.leadowner.php (hunk 1 above)
Comment (karuna835) [reply app/apis/webapi/backend/dash/get.leadowner.php]: added costant

File: app/apis/webapi/backend/dash/package.query.php (hunk 2 above)
Comment (karuna835) [reply app/apis/webapi/backend/dash/package.query.php]: added checks

File: app/templates/jaws/backend/dash/assets/js/controllers/payment.edit.js (hunk 3 above)
Comment (karuna835) [reply app/templates/jaws/backend/dash/assets/js/controllers/payment.edit.js]: added empty checks and added key to lead owner array

File: app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js (hunk 5 above)
Comment (karuna835) [reply app/templates/jaws/backend/dash/assets/js/controllers/payment.track.js]: added empty checks and made array key value pair

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**